### PR TITLE
Updates references from master to main

### PR DESCRIPTION
**Pull request by tabathad**
This PR is preparing to set this repository to create and use `main` as the default branch as a more inclusive term than the current default.

### CODE REVIEW

**Comment by mariannovotny** [comment]
Hi @tabathad Thanks for your PR. This PR should be discussed and synchronized with @jgiannuzzi in order to make our CI/CD pipeline functional because we are using GAEV currently set with 'master' branch.

File: pom.xml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (jgiannuzzi) [inline pom.xml:3]: Nitpick: we should probably leave this formatting change out of this PR

**Review by jgiannuzzi — COMMENTED** [review]
This is awesome, thanks @tabathad!

I believe however that this PR might be better off from a different branch than `main`, as it is currently preventing the rename. Maybe `inclusivity_ftw`? 😄

According to https://github.com/github/renaming, our branch protection settings should be updated automatically when we will rename the branch. However, I don't see any mentions of GitHub Actions Environments 🤔

Here is the plan I suggest we follow (I will be able to carry those steps):
* approve this PR
* rename `master` to `main`
* validate branch protection setting for `main`
* update environment access setting from `master` to `main`
* merge this PR -> this should trigger a new successful snapshot release

File: pom.xml (hunk 1 above)
Comment (tabathad) [reply pom.xml:3]: Whoops, unexpected formatting reverted in the new PR. Thanks!

**Comment by tabathad** [comment]
I've renamed this branch from 'main' to 'inclusivity_ftw' and reverted the formatting change. There doesn't seem to be a way to rename the branch once a PR is open so I'll close this PR in favor of the new one with the correct branch name, then delete the `main` branch so it can be used in renaming `master`. The new PR is here https://github.com/G-Research/siembol/pull/425

**Pull request closed without merging**